### ISSUE TITLE
'otherContent' is lost when converting from v2 to v3

### ISSUE THREAD

**Issue by jbaiter** [IIIF-Commons/parser#11]
I tried to convert a [V2 manifest from Europeana](https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/manifest) into v3 with `convertPresentation2`.
While this mostly works, there is one part that gets lost in translation, namely the annotation list that is referenced from `otherContent` on some canvases:

```json
{
  "@type": "sc:Canvas",
  "@id": "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/canvas/p21",
  "label": "p. 21",
  "height": 2048,
  "width": 1256,
  "attribution": "Journal historique et littéraire - 1788-09-01 - https://www.europeana.eu/item/9200396/BibliographicResource_3000118436165. National Library of Luxembourg. Public Domain Mark - http://creativecommons.org/publicdomain/mark/1.0/",
  "images": [
    {
      "@type": "oa:Annotation",
      "motivation": "sc:painting",
      "resource": {
        "@type": "dctypes:Image",
        "@id": "https://iiif.europeana.eu/image/QNYVL2Z2FVHRGMK2UNXIP4DUUOTOVA2ND3WIPJQF6V23SO2CJ5UA/presentation_images/ee0edfa0-0220-11e6-a696-fa163e2dd531/node-3/image/BNL/Journal_historique_et_littéraire/1788/09/01/00021/full/full/0/default.jpg",
        "format": "image/jpeg",
        "service": {
          "@context": "http://iiif.io/api/image/2/context.json",
          "@id": "https://iiif.europeana.eu/image/QNYVL2Z2FVHRGMK2UNXIP4DUUOTOVA2ND3WIPJQF6V23SO2CJ5UA/presentation_images/ee0edfa0-0220-11e6-a696-fa163e2dd531/node-3/image/BNL/Journal_historique_et_littéraire/1788/09/01/00021",
          "profile": "http://iiif.io/api/image/2/level1.json"
        }
      },
      "on": "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/canvas/p21"
    }
  ],
  "otherContent": [
    "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/annopage/21"
  ]
}
```

The converted v3 Canvas looks like this:

```json
{
  "id": "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/canvas/p21",
  "type": "Canvas",
  "height": 2048,
  "width": 1256,
  "label": {
    "none": [
      "p. 21"
    ]
  },
  "requiredStatement": {
    "label": {
      "none": [
        "Attribution"
      ]
    },
    "value": {
      "none": [
        "Journal historique et littéraire - 1788-09-01 - https://www.europeana.eu/item/9200396/BibliographicResource_3000118436165. National Library of Luxembourg. Public Domain Mark - http://creativecommons.org/publicdomain/mark/1.0/"
      ]
    }
  },
  "items": [
    {
      "id": "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/canvas/p21/annotation-page",
      "type": "AnnotationPage",
      "items": [
        {
          "id": "http://example.org/oa:Annotation/41",
          "type": "Annotation",
          "motivation": "painting",
          "target": "https://iiif.europeana.eu/presentation/9200396/BibliographicResource_3000118436165/canvas/p21",
          "body": {
            "id": "https://iiif.europeana.eu/image/QNYVL2Z2FVHRGMK2UNXIP4DUUOTOVA2ND3WIPJQF6V23SO2CJ5UA/presentation_images/ee0edfa0-0220-11e6-a696-fa163e2dd531/node-3/image/BNL/Journal_historique_et_littéraire/1788/09/01/00021/full/full/0/default.jpg",
            "type": "Image",
            "format": "image/jpeg",
            "service": [
              {
                "@id": "https://iiif.europeana.eu/image/QNYVL2Z2FVHRGMK2UNXIP4DUUOTOVA2ND3WIPJQF6V23SO2CJ5UA/presentation_images/ee0edfa0-0220-11e6-a696-fa163e2dd531/node-3/image/BNL/Journal_historique_et_littéraire/1788/09/01/00021",
                "@type": "ImageService2",
                "profile": "level2"
              }
            ]
          }
        }
      ]
    }
  ]
}
```

From my understanding of the v3 spec, there should be an `annotations` key with an array containing a single entry pointing to the external AnnotationPage..

**Comment by stephenwf** [comment]
Thanks @jbaiter good spot! Fixed in `v1.0.9`

**Comment by jbaiter** [comment]
Blazingly fast, thank you so much!!
